src/pages/components/itemCart/itemCart.ts

```diff
@@ -0,0 +1,383 @@
+import Promocode from '../../templates/promocode';
+import { getArrCart, removeItemFromCart, saveLocalStorage, updateHeader } from '../../../helpers/appFunctions';
+import iconsSVG from '../../templates/icons';
+import { Form } from '../form/form';
+
+const promoList: Promocode[] = [];
+promoList.push(new Promocode(promoList.length, 'rs', 10));
+promoList.push(new Promocode(promoList.length, 'epm', 15));
+
+function appendChildElements(parentElem: HTMLElement, arr: HTMLElement[]): void {
+    arr.forEach((el) => {
+        parentElem.appendChild(el);
+    });
+}
+
+//update total price in receint
+function updateTotalPrice(): void {
+    const totalPrice = <HTMLElement>document.querySelector('.receipt__total-price');
+    if (!totalPrice.classList.contains('used-promo')) totalPrice.classList.add('used-promo');
+    const promoPrice = <HTMLElement>document.querySelector('.receipt__total-promo');
+    if (promoPrice.classList.contains('hidden')) promoPrice.classList.remove('hidden');
```

Comment:
You can create as much elements as you want, but select something what's already existed in the DOM as few times as possible, ideal would be to use it just one time.